packages/client/wallets/aa/src/blockchain/wallets/CustomBridge.ts

```diff
@@ -0,0 +1,186 @@
+import { ZeroDevAccountSigner, ZeroDevEthersProvider } from "@zerodev/sdk";
+import { ethers } from "ethers";
+export class CustomEip1193Bridge {
+    readonly signer: ZeroDevAccountSigner<"ECDSA">;
+    readonly provider: ZeroDevEthersProvider<"ECDSA">;
+
+    constructor(signer:ZeroDevAccountSigner<"ECDSA">, provider: ZeroDevEthersProvider<"ECDSA">) {
```

Comment:
prettier not running it seems
```suggestion
    constructor(signer: ZeroDevAccountSigner<"ECDSA">, provider: ZeroDevEthersProvider<"ECDSA">) {
```